merp_outgoing_routing/tests/test_merp_outgoing_routing.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+from odoo.tests.common import TransactionCase
+from datetime import datetime
+
+
+class TestMerpOutgoingRouting(TransactionCase):
+
+    def setUp(self):
+        super(TestMerpOutgoingRouting, self).setUp()
+        self.user = self.env['res.users'].create({
+            'name': 'test_user',
+            'login': 'test_user'
+        })
+        self.ventor_worker = self.env.ref('merp_custom_access_rights.ventor_role_wh_worker')
+        self.ventor_worker.write({'users': [(4, self.user.id, 0)]})
```

Comment:
s the third param (0) really needed here? Should'nt it be `self.ventor_worker.write({'users': [(4, self.user.id)]})`?